bubblesort.rb

```diff
@@ -0,0 +1,15 @@
+def bubble_sort(arr)
+  len = arr.length - 1
+  while len
+    unsorted = false
+    (0...len).each do |i|
+      if arr[i] > arr[i + 1]
+        arr[i + 1], arr[i] = arr[i], arr[i + 1]
+        unsorted = true
+      end
+    end
+    break if unsorted == false end
```

Comment:
- [ ] The `end` keyword should be in the next line or line 12
- [ ] Keep code DRY, refactor the `break if unsorted == false` for one of these options `break unless unsorted` or `break if !unsorted`